### PR TITLE
Added endpoint for returning details of single product.

### PR DESCRIPTION
**Pull request by arjunrn**
I'm working on the report generation at the moment. In order to generate better report indexes it would be useful to have the complete product names. I've added this endpoint so that these details can be fetched. It's also a part of the new API schema.

### CODE REVIEW

File: app/handler/product.py [hunk 1]
```diff
@@ -27,6 +27,18 @@ def get(pg=None):
...
 
 
+def get_product(product):
+    with dbconn() as conn:
+        cur = conn.cursor()
+        cur.execute('''SELECT p_product_group_id, p_name, p_slug, p_delivery_team
+            FROM zsm_data.product WHERE p_slug = %s''', (product,))
+        rows = cur.fetchall()
+        if len(rows) < 1:
+            return NoContent, 404
```
Comment (mohabusama) [inline app/handler/product.py]: May be returning Problem with 404 would be more informative to the user.

**Comment by mohabusama** [comment]
👍

**Comment by mohabusama** [comment]
👍

**Comment by apfeiffer85** [comment]
👍

**Pull request merged**